File: 01 - Iniciante/1065 - Pares entre cinco numeros/problem1065.cpp
/*  
	URI Online Judge 
	Problema 1065 - Pares entre Cinco Números
    https://www.urionlinejudge.com.br/repository/UOJ_1065.html
*/

#include <iostream>
 
using namespace std;
 
int main() {
    int num, par = 0;
 
    for (int i=0; i<5; i++) {
        cin >> num;
        if (num % 2 == 0) {
            par++;
        }
    }
    
    cout << par << " valores pares" << endl;
 
    return 0;
}

File: 01 - Iniciante/1065 - Pares entre cinco numeros/problem1065.py
""" 
	URI Online Judge 
	Problema 1065 - Pares entre Cinco Números
    https://www.urionlinejudge.com.br/repository/UOJ_1065.html 
"""
par = 0
for i in range(5):
    num = int(input())
    if num % 2 == 0:
        par += 1

print '%i valores pares' % par
